apps/generator/Dockerfile

```diff
@@ -11,9 +11,11 @@ ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
 # Since 0.30.0 release Git is supported and required as a dependency
 # Since 0.14.0 release of html-template chromium is needed for pdf generation.
 # More custom packages for specific template should not be added to this dockerfile. Instead, we should come up with some extensibility solution.
-RUN apk --update add git chromium && \
-    rm -rf /var/lib/apt/lists/* && \
-    rm /var/cache/apk/*
+#is the following code correct for removing the cache after installing packages or stor it in cache mount?
```

Comment:
you are trying to fix the sonarcloud issue or ask question by this comment?

Reply:
Its a solution im trying to provide.
And sorry for that Comment 😐,  pl ignore it, I will remove it